components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/connector/recaptcha/SSOLoginReCaptchaConfig.java

```diff
@@ -130,18 +141,26 @@ public CaptchaPreValidationResponse preValidate(ServletRequest servletRequest, S
             preValidationResponse.setCaptchaAttributes(params);
             preValidationResponse.setOnCaptchaFailRedirectUrls(CaptchaUtil.getOnFailedLoginUrls());
             preValidationResponse.setCaptchaValidationRequired(true);
-
-        } else if (CaptchaUtil.isMaximumFailedLoginAttemptsReached(MultitenantUtils.getTenantAwareUsername(username),
-                tenantDomain)) {
-            preValidationResponse.setCaptchaValidationRequired(true);
-            preValidationResponse.setMaxFailedLimitReached(true);
-
-            preValidationResponse.setOnCaptchaFailRedirectUrls(CaptchaUtil.getOnFailedLoginUrls());
-            Map<String, String> params = new HashMap<>();
-            params.put(CaptchaConstants.RE_CAPTCHA, CaptchaConstants.TRUE);
-            params.put(CaptchaConstants.AUTH_FAILURE, CaptchaConstants.TRUE);
-            params.put(CaptchaConstants.AUTH_FAILURE_MSG, CaptchaConstants.RECAPTCHA_FAIL_MSG_KEY);
-            preValidationResponse.setCaptchaAttributes(params);
+        } else {
+            String failedLoginAttemptsClaim = FAILED_LOGIN_ATTEMPTS_CLAIM_URI;
```

Comment:
let's indicate its the URI in the variable name as well for better readability

Reply:
Changed the variable name